src/plugins/vis_types/timeseries/server/lib/vis_data/helpers/get_splits.ts

```diff
@@ -25,6 +26,8 @@ interface SplittedData<TMeta extends BaseMeta = BaseMeta> {
   id: string;
   splitByLabel: string;
   label: string;
+  labelFormatted?: string;
+  termsSplitValue?: string | MultiFieldKey;
```

Comment:
Not sure that you can use `MultiFieldKey` type here. In fact from the Server side you pass serialized type of that object.  In future it can confuse someone.